server/controllers/skillSet.js

```diff
@@ -0,0 +1,47 @@
+const SkillSet = require('../models/skillSet');
+
+const getSkillSet = async (req, res, next) => {
+    try {
+        const skillSet = await SkillSet.find();
```

Comment:
@JordanJamesTaylor let's change this to say `skillSets` since we are getting back more than one.